quipucords/fingerprinter/__init__.py

```diff
@@ -24,23 +24,11 @@
 NETWORK_IDENTIFICATION_KEYS = ['subscription_manager_id',
                                'bios_uuid']
 VCENTER_IDENTIFICATION_KEYS = ['vm_uuid']
-COMMON_FACTS_TO_MERGE = ['os_name',
-                         'os_version',
-                         'os_release',
-                         'mac_addresses',
-                         'ip_addresses',
-                         'cpu_count']
-VCENTER_FACTS_TO_MERGE = ['vm_name',
-                          'vm_state',
-                          'vm_uuid',
-                          'vm_memory_size',
-                          'vm_dns_name',
-                          'vm_host',
-                          'vm_host_cpu_cores',
-                          'vm_host_cpu_threads',
-                          'vm_host_socket_count',
-                          'vm_datacenter',
-                          'vm_cluster']
+NETWORK_SATELLITE_MERGE_KEYS = [
+    ('subscription_manager_id', 'subscription_manager_id')]
```

Comment:
Should be able to merge on mac_addresses too.

Reply:
Added support to dedup on mac_addresses.